src/blocks/mrc_class_method_def.ts

```diff
@@ -609,3 +617,31 @@ export function getMethodNamesAlreadyOverriddenInWorkspace(
     }
   });
 }
+
+/**
+ * Upgrades the ClassMethodDefBlocks in the given workspace from version 002 to 003.
+ * This function should only be called when upgrading old projects.
+ */
+export function upgrade_002_to_003(workspace: Blockly.Workspace): void {
+  // Make sure the workspace is headless.
+  if (workspace.rendered) {
```

Comment:
Should we pull these out and check it in the upgrade project so it doesn't have to be littered throughout the code?

Reply:
The purpose of checking it here is to make sure this function is not called from code other than the upgrade code. But, since these functions all have names upgrade_*_to_*, I guess it is not necessary.